wordpress-mobile/WordPress-iOS
# localize.py: Surround arguments with quotes.

Fixes #10051 

This PR solves the issue where none of the strings in the Stock Photos sections was localized, even when all the strings were properly handled in code.

After performing some tests, I noticed that the problem was the space in the name of the folder `Stock Photos`.

The `localize.py` script sends the filename arguments separated by spaces, so a file that is inside a folder with a space in its name will be mistakenly recognized as two different and **nonexistent** files.

i.e.: `Classes/ViewRelated/Media/Stock Photos/StockPhotosStrings.swift`

To avoid this issue in the future, the solution was to wrap up all file names with quotes.

To test:
- Run the script locally `python Scripts/localize.py`
- Open `WordPress/Resources/en.lproj/Localizable.strings`
- Check that the key `"Search to find free photos to add to your Media Library!"` exists.
